MagneticField/ParametrizedEngine/interface/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,47 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_ParabolicParametrizedMagneticField_h
+
+namespace magneticFieldParabolicPortable {
+
+  struct Parameters {
+    // These parameters are the best fit of 3.8T to the OAEParametrizedMagneticField parametrization.
+    // See MagneticField/ParametrizedEngine/src/ParabolicParametrizedMagneticField.cc
+    static constexpr float c1 = 3.8114;
+    static constexpr float b0 = -3.94991e-06;
+    static constexpr float b1 = 7.53701e-06;
+    static constexpr float a = 2.43878e-11;
+    static constexpr float max_radius2 = 13225.f;  // tracker radius
+    static constexpr float max_z = 280.f;          // tracker z
+  };
+
+  template <typename Vec3>
+  constexpr float Kr(Vec3 const& vec) {
```

Comment:
It might be useful to add a comment somewhere in this file what kind of interface is expected from `Vec3` (even if it is mostly clear from the implementation), and note that e.g. `Eigen::Matrix` provides that interface.

Reply:
Added: https://github.com/cms-sw/cmssw/pull/44360/commits/a1478e86510ac42071a06a1bb70fcb6325b2417a